specs/UpdateRuntime.md

```diff
@@ -0,0 +1,229 @@
+                wil::com_ptr<ICoreWebView2Environment> webViewEnvironment = environment;
+                auto experimentalEnvironment3 = webViewEnvironment.try_query<ICoreWebView2ExperimentalEnvironment3>();
+                HRESULT hr = experimentalEnvironment3->UpdateRuntime(
+                    Callback<ICoreWebView2ExperimentalUpdateRuntimeCompletedHandler>(
+                        [callback, minimalVersionRequired, experimentalEnvironment3](HRESULT errorCode,
+                           ICoreWebView2ExperimentalUpdateRuntimeResult* result) -> HRESULT {
+                            COREWEBVIEW2_RUNTIME_UPDATE_STATUS updateStatus =
+                                COREWEBVIEW2_RUNTIME_UPDATE_STATUS_FAILED;
+                            if ((errorCode == S_OK) && result)
+                            {
+                              CHECK_FAILURE(result->get_UpdateRuntimeStatus(&updateStatus));
+                            }
+                            if (updateStatus != COREWEBVIEW2_UPDATE_RUNTIME_STATUS_LATEST_VERSION_INSTALLED)
+                            {
+                              callback(false);
+                            }
+                            else
+                            {
+                              callback(IsCurrentVersionSameOrNewer(minimalVersionRequired));
+                            }
+                            return S_OK;
+                        })
+                        .Get());
+                return S_OK;
+            })
+            .Get());
+}
+
+```
+
+# Remarks
+See comments in [API Details](#api-details) section below.
+
+# API Notes
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+
+## Win32 C++
+```IDL
+/// Status of UpdateRuntime operation result.
+[v1_enum] typedef enum COREWEBVIEW2_UPDATE_RUNTIME_STATUS {
+
+  /// Latest version of Edge WebView2 Runtime is installed.
+  /// No update for Edge WebView2 Runtime is available, or Edge WebView2
+  /// Runtime is updated successfully and latest version is now installed.
+  COREWEBVIEW2_UPDATE_RUNTIME_STATUS_LATEST_VERSION_INSTALLED,
+
+  /// Edge WebView2 Runtime update is blocked by group policy.
+  COREWEBVIEW2_UPDATE_RUNTIME_STATUS_BLOCKED_BY_POLICY,
+
+  /// Edge WebView2 Runtime update failed.
+  /// See `UpdateError` property of UpdateRuntimeResult for more
+  /// information about the failure.
+  COREWEBVIEW2_UPDATE_RUNTIME_STATUS_FAILED,
+} COREWEBVIEW2_UPDATE_RUNTIME_STATUS;
+
+
+/// The UpdateRuntime operation result.
+[uuid(DD503E49-AB19-47C0-B2AD-6DDD09CC3E3A), object, pointer_default(unique)]
+interface ICoreWebView2ExperimentalUpdateRuntimeResult : IUnknown {
+
+  /// The status for the UpdateRuntime operation.
+  [propget] HRESULT Status(
+      [ out, retval ] COREWEBVIEW2_UPDATE_RUNTIME_STATUS * status);
+
+  /// The update error happened while trying to update Edge WebView2 Runtime.
```

Comment:
Only valid if Status is Failed?

Reply:
Need to define the value in documentation for success